app/api/routes.py

```diff
@@ -437,3 +435,22 @@ def validate_resource(json):
 
     if validation_errors['errors']['missing_params']:
         return validation_errors
+
+
+def format_resource_search(hit):
```

Comment:
This file is super long, let's go ahead and stick this in `utils`. We should go ahead and stick `validate_resource` and `create_new_apikey` in resources too while we're at it